packages/babel-plugin-transform-runtime/src/definitions.js

```diff
@@ -1,3 +1,12 @@
+import semver from "semver";
+import { version } from "@babel/core";
```

Comment:
This isn't the correct version in this context. We'll need to use the `version` passed into the plugin itself in https://github.com/babel/babel/blob/53e4963836a89141845261029d332475668aa26b/packages/babel-plugin-transform-runtime/src/index.js#L40

Reply:
Is `runtimeVersion` a global variable?

Reply:
It is not, so either we'd have to do this check in `index.js` instead, or we'd have to change `export default` here to export a function that passes in the version.